utils/utils.go

```diff
@@ -256,3 +257,34 @@ func ValidateRawGithubUrl(url string) error {
 
 	return nil
 }
+
+func TypedErr(e interface{}) error {
+	switch t := e.(type) {
+	case error:
+		return t
+	case string:
+		return errors.New(t)
+	default:
+		return nil
+	}
+}
+
+func WrapError(mainErr interface{}, subErr interface{}) error {
+	var main, sub error
+	main = TypedErr(mainErr)
```

Comment:
can you move all of these to utils/errors.go instead?

Reply:
> Just done!